samples/QR-app-installation/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,49 @@
+{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.10/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.10",
+  "version": "1.0.0",
+  "id": "{{MicrosoftAppId}}",
+  "packageName": "com.microsoft.teams.qrappinstallation",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "QRAppInstallation",
+    "full": "Install app by QR"
+  },
+  "description": {
+    "short": "Install app by QR",
+    "full": "This sample demos app installation using QR code which has application app id"
+  },
+  "icons": {
+    "outline": "outline.png",
+    "color": "color.png"
+  },
+  "accentColor": "#60A18E",
+  "bots": [
+    {
+      "botId": "87a133ac-47f2-4d47-9af2-b9115a5edbf3",
+      "scopes": [
+        "team",
+        "personal",
+        "groupchat"
+      ],
+      "isNotificationOnly": false
+    }
+  ],
+  "permissions": [ "identity", "messageTeamMembers" ],
+  "devicePermissions": [
+    "media"
+  ],
+  "validDomains": [
+    "*.ngrok.io",
```

Comment:
In this sample you have mentioned .ngrok as valid domains, please check in C# example where you have added dummy data.